dGame/dGameMessages/GameMessages.cpp

```diff
@@ -4977,7 +4977,12 @@ void GameMessages::HandlePlayEmote(RakNet::BitStream& inStream, Entity* entity)
 		if (emote) sAnimationName = emote->animationName;
 	}
 
-	RenderComponent::PlayAnimation(entity, sAnimationName);
+	//RenderComponent::PlayAnimation(entity, sAnimationName);
+	GameMessages::EmotePlayed msg;
+	msg.target = entity->GetObjectID();
+	msg.emoteID = static_cast<int32_t>(emoteID);
```

Comment:
instead of a static cast here, the `int` type of emoteID should be changed out with `int32_t`.  on most platforms `int` does refer to a 32 bit int, however theres always the odd chance we find a platform with 16 bit ints.  For DLU's case since we are a cross platform server for a windows compiled game, we always use specific bit widths for bitstreams.  

As an example, heres a godbolt snippet of why we do this https://godbolt.org/z/svosssbvf